packages/specification/electronic-prescription-service-api.yaml

```diff
@@ -417,6 +422,11 @@ paths:
                   description: A successful response to an acute, primary care prescription order request.
                   value:
                     $ref: examples/primary-care/acute/nominated-pharmacy/medical-prescriber/1-Process-Response-Send-200_OK.json
+                acute-multiple-dosages:
+                  summary: Acute
```

Comment:
Please refrain from using identical names for different examples. 
Please indicate in the name that the example pertains to multiple dosage items.

Reply:
This is not identical. It expresses that it contains multiple dosages. I could add more info in the summary but I won't change the title. Thank you.